### PR TITLE
RHVH / oVirt Node provisioning templates

### PR DESCRIPTION
**Pull request by lzap**
Templates for oVirt Node NG or RHVH provisioning. Tested with RHVH 4.1.

https://www.youtube.com/watch?v=L6IvTkjhSx8

### CODE REVIEW

File: provisioning_templates/provision/kickstart_ovirt.erb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- ntp-server: string (default="0.fedora.pool.ntp.org")
+- disable-firewall: boolean (default=false)
+
+This kickstart will only work with LVM THIN partitioning ('Kickstart default thin')
+and it requires the installation URL to have squashfs.img image extracted in the
+root folder (or specified via 'liveimg_name' parameter). See oVirt Node documentation
+or RHV Installation Manual, section 5.2. Advanced installation.
+%>
+install
+liveimg --url=<%= @host.operatingsystem.medium_uri(@host) %>/<%= @host.params['liveimg_name'] || 'squashfs.img' %>
```
Comment (ik5) [inline provisioning_templates/provision/kickstart_ovirt.erb:25]: Why not use `File.join`?
e.g. 
```
<%= File.join(@host.operatingsystem.medium_uri(@host), @host.params['liveimg_name'] || 'squashfs.img') %>
```

I think it is a bit more readable that way, wdyt?

**Review by ik5 — COMMENTED** [review]
just small question

File: provisioning_templates/provision/kickstart_ovirt.erb (hunk 1 above)
Comment (ekohl) [reply provisioning_templates/provision/kickstart_ovirt.erb:25]: Because URLs are not file paths. It'd break on Windows.

**Comment by mmoll** [comment]
untested, but looks good visually :+1:

**Comment by mmoll** [comment]
merged, díky @lzap!

**Pull request merged**